README.md

```diff
@@ -2,197 +2,68 @@
-[Releases](https://github.com/lightninglabs/lightning-terminal/releases) page in this
-repo. There you can download the latest version and extract the archive into a directory
-on your computer.
-
-#### Compile from Source Code
-
-To compile from source code, you'll need to have some prerequisite developer tooling
-installed on your machine.
-
-| Dependency                                          | Description                                                                                                                                                                          |
-| --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| [golang](https://golang.org/doc/install)            | LiT's backend web server is written in Go. The minimum version supported is Go v1.13.                                                                                                |
-| [protoc](https://grpc.io/docs/protoc-installation/) | Required to compile LND & Loop gRPC proto files at build time                                                                                                                        |
-| [nodejs](https://nodejs.org/en/download/)           | LiT's frontend is written in TypeScript and built on top of the React JS web framework. To bundle the assets into Javascript & CSS compatible with web browsers, NodeJS is required. |
-| [yarn](https://classic.yarnpkg.com/en/docs/install) | a popular package manager for NodeJS application dependencies                                                                                                                        |
-
-Once you have the necessary prerequisites, LiT can be compiled by running the following
-commands:
+## Execution
+Run Lightning Terminal with a local LND instance:
 
 ```
-git clone https://github.com/lightninglabs/lightning-terminal.git
-cd lightning-terminal
-make install
+./litd --uipassword=UP48lm4VjqxmOxB9X9stry6VTKBRQI
 ```
 
-This will produce the `litd` executable and add it to your `GOPATH`. The CLI binaries for
-`lncli`, `loop`, and `frcli` are not created by `make install`. You will need to download
-those binaries from the [lnd](https://github.com/lightningnetwork/lnd/releases),
-[loop](https://github.com/lightninglabs/loop/releases), and
-[faraday](https://github.com/lightninglabs/faraday/releases) repos manually.
-
-#### Executing CLI Commands
+Visit https://localhost:8443 to access Terminal.
 
-When executing `loop` and `frcli` commands, you will need to specify the connection info
-since the daemons are now integrated into `lnd`'s GRPC server.
+Note that a password with a minimum of 8 characters is required to run Lightning Terminal. In a production environment, it's recommended that you store this password as an environment variable.
```

Comment:
Add "... variable to avoid it being recorded in the command history"?

Reply:
Fixed